### PR TITLE
[Australia}: extend CFP date by a week

### PR DESCRIPTION
**Pull request by swapnilogale**
@plaindocs extending CFP by a week. Updated dates and announcement.

----
:books: Documentation preview :books:: https://writethedocs-www--2030.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/australia/2023/news/cfp-extend.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+===========================
+
+Greetings fellow documentarians!
+
+The conference is now just under three months away, and we hope you’re already planning for this year's onsite installment of Write the Docs Australia conference.
+
+New deadline to submit your proposal is 22nd Sept
+--------------------------------------------------
+
+We added an extra week to the deadline because we wanted to give as many of you the opportunity to prepare your submissions.
```
Comment (plaindocs) [inline docs/conf/australia/2023/news/cfp-extend.rst]: Missing words?

File: docs/conf/australia/2023/news/cfp-extend.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Make sure to get your talk in before the new deadline; we already have a lot of great proposals coming in, but we wanted to give everyone a chance to submit their ideas.
```
Comment (plaindocs) [inline docs/conf/australia/2023/news/cfp-extend.rst]: Repeated words?

File: docs/conf/australia/2023/opportunity-grants.rst [hunk 3]
```diff
@@ -33,7 +32,7 @@ Grant applicants, like all other participants in the Write the Documents communi
 What is covered
 ----------------
 
-All grants include a free conference ticket for the virtual conference.
+All grants include a free conference ticket for the onsite conference.
```
Comment (plaindocs) [inline docs/conf/australia/2023/opportunity-grants.rst]: No travel or accommodation?

**Review by plaindocs — CHANGES_REQUESTED** [review]
Few things to fix. :+1:

**Comment by swapnilogale** [comment]
> Few things to fix. 👍

Fixed now @plaindocs

File: docs/conf/australia/2023/opportunity-grants.rst [hunk 4]
```diff
@@ -18,7 +18,7 @@ level or background.
 
 Depending on the number of applications, we may not be able to provide every applicant with a free ticket. We prioritize applications based on the overall impact that granting an application will have on the applicant, the Write the Docs community, and the applicant's wider community and country, compared to others.
 
-Grant applicants, like all other participants in the Write the Documents community, are required to conform to the Code of Conduct: https://www.writethedocs.org/code-of-conduct/.
+Grant applicants, like all other participants in the Write the Docs community, are required to conform to the Code of Conduct: https://www.writethedocs.org/code-of-conduct/.
```
Comment (plaindocs) [inline docs/conf/australia/2023/opportunity-grants.rst:21]: :sunglasses:

**Review by plaindocs — APPROVED** [review]

File: docs/conf/australia/2023/opportunity-grants.rst (hunk 4 above)
Comment (swapnilogale) [reply docs/conf/australia/2023/opportunity-grants.rst:21]: Fair point though :P

**Pull request merged**